packages/terra-date-picker/src/react-datepicker/date_utils.js

```diff
@@ -1,5 +1,7 @@
 import moment from 'moment'
 
+moment.suppressDeprecationWarnings = true;
```

Comment:
Why has this been included in src code? Is this something we need to disabled in tests instead? Do we need to log an issue to remove this later on?

Reply:
This can be removed. I missed taking that out from diagnosing a previous issue. :/